.travis.yml

```diff
@@ -14,9 +14,10 @@ before_install:
 install:
   - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION
   - conda activate test-env
-  - conda install -q -c conda-forge biom-format flake8 coveralls
+  - conda install -q -c conda-forge biom-format flake8 nose coveralls
   - python setup.py install
 script:
-  - make all
+  - flake8
```

Comment:
```suggestion
  - flake8 woltka/ setup.py
```

Reply:
Hmmm... Will `flake8 setup.py` also take care of `woltka/*.py`, `woltka/tests/*.py`, `woltka/q2/*.py`, etc.? I was trying to make it general for the entire package regardless of structure.

Reply:
Away from computer, but after reading the flake8 docs some more, it looks like invoking
flake8
Like you did will lint all *.py files in the project. I guess this is fine, it just seems to be lab convention to narrow it to the main python module and setup.py
Also,
flake8 setup.py
Will only lint the setup.py
Can you just verify for me that using flake8 the way you did can indeed catch pep8 violations? I’m not sure about the _best_ way to check that flake8 is linting the files you expect.... but one thing to try is creating some directories that contain intentional pep8 violations and seeing if it fails or not.

Reply:
I have been using `flake8` at all times. I am very certain that it will catch all Python scripts under the current directory, even if those are not part of the package (i.e., subdirectories have no `__init__.py`). PS: I guess you meant `pycodestyle`, to which `pep8` has been renamed.

Reply:
Also FYI: `python -m unittest` needs `__init__.py`, whereas `nosetests` doesn't. :)

Reply:
Okay great!